src/script/auth/component/ProgressBar.tsx

```diff
@@ -46,7 +46,6 @@ export const ProgressBar = ({width, percent, error, style}: ProgressProps) => {
           backgroundColor: error ? 'red' : 'blue',
           borderRadius: '5px',
           height: 'inherit',
-          transition: '0.5s ease-in',
```

Comment:
i think this makes it appear nicer also, id probably leave this

Reply:
With the transition, the change is horribly delayed, and the progress bar still moves for some time after the mouse movement was stopped. This is probably the fault of the many small incremental updates. Maybe there is a better solution, so I'm definitely open to suggestions.

Reply:
maybe ill play around with this a bit and see if there's some better solution